code/modules/clothing/head/berets.dm

```diff
@@ -198,12 +188,6 @@
 
 //centcom
 
-/obj/item/clothing/head/beret/centcom/liaison
-	name = "corporate liaison beret"
-	desc = "A stylish beret worn by corporate liaisons."
-	icon_state = "centcomofficer"
-	item_state = "centcomofficer"
```

Comment:
What do CCIA outfits use now if you've removed all of this stuff?

Reply:
CCIA outfits actually use /obj/item/clothing/head/beret/centcom/officer

this was a placeholder for corporate liaisons